lib/Option/SanitizerOptions.cpp

```diff
@@ -140,7 +139,7 @@ SanitizerKind swift::parseSanitizerArgValues(const llvm::opt::Arg *A,
   // Thread Sanitizer only works on OS X and the simulators. It's only supported
```

Comment:
Thanks. Just wanted to let you know I am working on this. Once I figure it out, I'll be back with updates.